codecparsers/mpeg2_parser.cpp

```diff
@@ -568,36 +646,40 @@ namespace MPEG2 {
         bitReaderReadBits(10, &(m_sequenceHdr.vbv_buffer_size_value));
         bitReaderReadFlag(&(m_sequenceHdr.constrained_params_flag));
 
-        bitReaderReadFlag(&(m_sequenceHdr.load_intra_quantiser_matrix));
+        bitReaderReadFlag(&(quantMatrices->load_intra_quantiser_matrix));
```

Comment:
better use following function to remove duplication
`loadMatrix(bool& loaded, uint8_t matrix[64], uint8_t defaultMatrix[64])`
and this function can call 
`loadMatrix(bool& loaded, uint8_t matrix[64])`
to remove duplicate code further